### PR TITLE
Complete 0.2.0 changelog + site consistency

### PR DESCRIPTION
**Pull request by arthurvd**
* missing topics/tutorials in nav
* Changelog
* mkdocs-macros for easy linking to GitHub and User Manuals.
* list of funtionalities find/replace 0.1.6* -> 0.2.0
* added HYDROLIB logo (but needs a white version)

### CODE REVIEW

File: docs/include/mkdocs-macros/main.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        "https://content.oss.deltares.nl/delft3d/manuals/D-Flow_FM_User_Manual_1D2D.pdf"
+    )
+    sobek_um_url = (
+        "https://content.oss.deltares.nl/delft3d/manuals/SOBEK_User_Manual.pdf"
+    )
+    hydrolib_core_url = "https://github.com/Deltares/HYDROLIB-core"
+
+    @env.macro
+    def mymacro():
+        return "Hello World"
```
Comment (evetion) [inline docs/include/mkdocs-macros/main.py:14]: Hello there! Maybe remove this tryout?

File: docs/include/mkdocs-macros/main.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
+def define_env(env):
```
Comment (evetion) [inline docs/include/mkdocs-macros/main.py:1]: Maybe a small explanation as module docstring about what these things do and how others can use them in the documentation?

**Review by evetion — APPROVED** [review]
That's quite a lot of documentation, well done!

File: docs/include/mkdocs-macros/main.py (hunk 2 above)
Comment (arthurvd) [reply docs/include/mkdocs-macros/main.py:1]: Done.

File: docs/include/mkdocs-macros/main.py (hunk 1 above)
Comment (arthurvd) [reply docs/include/mkdocs-macros/main.py]: done

**Pull request merged**